docs/pages/access-controls/access-lists.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Access Lists (Preview)
+description: Use and enforce trusted devices with Teleport
```

Comment:
We need to do a better job at not leaving copy-pasta behind. This is not a device trust guide.